packages/@webex/internal-plugin-voicea/test/unit/spec/voicea.js

```diff
@@ -69,6 +69,16 @@ describe('plugin-voicea', () => {
           trackingId: sinon.match.string,
         });
       });
+
+      it('listens to events once', () => {
+        assert.equal(voiceaService.hasSubscribedToEvents, false);
```

Comment:
hasSubscribedToEvents is private so we shouldn't be reading it in the test. I think you're only able to do it because this test file is a js file not ts - otherwise typescript would not let you read the private field. Anyway, we don't need to check that private field, checking the spy on listenToEvents is enough. Although you'll need to spy on `webex.internal.llm` now (see my other comment)

Reply:
Modified